app/reports/test_report.rb

```diff
@@ -83,8 +83,9 @@ def column_attrs
 
     attrs["Unique PI Last Name"] = :last_name
     attrs["Unique PI First Name"] = :first_name
-    attrs["College"] = [:college, COLLEGES.invert] # we invert since our hash is setup {"Bio Medical" => "bio_med"} for some crazy reason
-    attrs["Department"] = [:department, DEPARTMENTS.invert]
+    attrs["Institution"] = "try(:professional_organization).try(:parents_and_self).try(:select){|org| org.org_type == 'institution'}.try(:first).try(:name)"
+    attrs["College"] = "try(:professional_organization).try(:parents_and_self).try(:select){|org| org.org_type == 'college'}.try(:first).try(:name)"
+    attrs["Department"] = "try(:professional_organization).try(:parents_and_self).try(:select){|org| org.org_type == 'department'}.try(:first).try(:name)"
```

Comment:
It's the example report, so yeah. I only updated it in the first place to show how you COULD do such a call. So it's good like this.